common/source/docs/common-choosing-a-ground-station.rst

```diff
@@ -295,3 +295,36 @@ For Desktop devices you will need:
 
 -  A USB cable, BT module, WIFI adapter, or other connectivity means as listed here (:ref:`common-telemetry-landingpage`)
```

Comment:
This page does not exist in Sub, so I expect that link would be broken.

Reply:
I suspect that it actually may not be....locally, it will redirect to any wiki that has that tag...not sure that it will on the main server, but I am going to merge for a test....I can revert if it does not